fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -229,8 +205,7 @@ private void showWalletWarning() {
   }
 
   private void openSocialMediaTab() {
-    List<Lao> laos = mViewModel.getLAOs().getValue();
-    if (laos == null || laos.isEmpty()) {
+    if (!Boolean.TRUE.equals(viewModel.isSocialMediaEnabled().getValue())) {
```

Comment:
why do we work with Boolean objects instead of just boolean values ? Can't we avoid it ? 😅

Reply:
Because it is the generic of the livedata.
And for some reason, in java, a generic must be a class. Not a primitive.

That is why we cannot use boolean. And it comes with the cost that we have not warranty concerning the nullability of a Boolean.